test/lightning/usage_tracking/workflow_metrics_service_test.exs

```diff
@@ -1,5 +1,6 @@
 defmodule Lightning.UsageTracking.WorkflowMetricsServiceTest do
-  use Lightning.DataCase, async: true
+  # Async false becuase Mox returns nil whenit should return 100 for seed 976623
+  use Lightning.DataCase, async: false
```

Comment:
I tracked down the cause of this, it's because of the use of:

```
put_temporary_env(:lightning, :usage_tracking,
  cleartext_uuids_enabled: cleartext_uuids_enabled,
  enabled: true,
  host: @host
)
```

in `test/lightning/usage_tracking/report_worker_test.exs`.

Could you update those tests to use Mox (at least in the cases of `put_temporary_env`) instead.

Reply:
I think we're good here for now, but it is curious why codecov thinks those functions aren't being called though, because they are - or maybe it's because we are stubbing them?

Reply:
@stuartc Yeah, maybe it is because they are stubbed in all the other tests, whereas other methods have at least one test where they actually hit? That being said, I tried breaking some of the methods and no test breakage was observed, so CodeCov moves in mysterious ways...